### ISSUE TITLE
Major Bug in Ahn'Qiraj Temple

### ISSUE THREAD

**Issue by way2adept** [dalaranwow/dalaran-wow#5551]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: The ancient Silithid sandworm **Ouro** (boss) in Ahn'Qiraj Temple never comes out of the ground. It only displays out of ground surface for 1 second when approached for the first time, but immediately disappears the next second. It cannot be targeted/attacked or made to come out to surface again. The players need to kill him for getting Vanilla PVE Legs, which can only be acquired after submitting his skin a.k.a. **Skin of the Great Sandworm** along with some other items. 
Moreover, I tried doing A'Q temple several times for completing that very quest in particular, but each time the same problem arises.
**Ouro** hence is bugged, and need to be fixed soon.

**Expected behaviour**: The ancient Silithid sandworm **Ouro** should pop out to surface for the fight and must drop epic items along with his skin for those having the quest.

**Steps to reproduce the problem**:

1. Enter A'Q and reach the area where the boss **Ouro** resides.
2. Approach him in an attempt to engage him in combat.
3. You're done. Bugged **Ouro** is right in front of you.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
